code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -109,7 +109,7 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	var/icon_off = "cigoff"
 	var/type_butt = /obj/item/weapon/cigbutt
 	var/lastHolder = null
-	var/smoketime = 300
+	var/smoketime = 150

```

Comment:
The smoking time is intentionally and by design based on real cigs. 5 minutes is about how long it takes to smoke one.

The point being that you can have your character light up a smoke, go out and smoke, and come back and your character's smoke be done, as a way to signify why you are afk.

Better ways to balance anything relating to smoking time and reagents, would be to give cigarettes a bit of waste (since the filter would absorb a bunch of the reagents in the smoke, not all of the reagents have to be used), and nerf how much it can hold.